sdks/java/core/src/test/java/org/apache/beam/sdk/testing/TestStreamTest.java

```diff
@@ -415,7 +418,12 @@ public void testEarlyPanesOfWindow() {
   }
 
   @Test
-  @Category({ValidatesRunner.class, UsesTestStream.class, UsesTestStreamWithMultipleStages.class})
+  @Category({
+    ValidatesRunner.class,
+    UsesTestStream.class,
+    UsesTestStreamWithMultipleStages.class,
+    UsesStatefulParDo.class
```

Comment:
This small change should be an independent commit that comes first. Then if there is any trouble we will only revert the bigger commit but not this fixup.

Reply:
Done ✔️ 
https://github.com/apache/beam/pull/23202